### DISCUSSION TITLE
TTRPG Admin Repository

### DISCUSSION THREAD

**Discussion by sigrunixia** [Obsidian-TTRPG-Community/TTRPG-Community-Admin#9] [The Burned Steak]
**Current Status**: Setting Up Standards and Rules

**Comment by sigrunixia** [comment]
In the [releases branch](https://github.com/Obsidian-TTRPG-Community/TTRPG-Community-Admin/tree/releases), I have started building the "new to TTRPG Community" guide. 

Not quite sure which direction I am going in, so playing it by ear.

**Comment by sigrunixia** [comment]
@Obsidian-TTRPG-Community/inquisitors 

In preparation for June SRD projected, I wanted to get your thoughts on the current guidelines for SRD's and if they should be altered in any way. These would then be store in the Admin repo going forward. I'll post them in the first comment below.

Some things that are on my mind:
1. I'm about to calculate the votes, but there is a strong likelihood that XWN will be the winner. As such, we need to add a rule on rewording text when a publisher allows us to, and "Best Practices" around that.
2. I'm getting mixed feedback on Wikilink versus Markdown Links, and on Relevant, versus Short, versus Absolute. Much of the feedback is coming from "newbies", particularly those with windows systems. I have no desire to leave anyone high and dry on any OS, but would like more eyes on this as I had initially created these rules with no input from others.
3. Particularly, due to XWN and my Bunkers and Badasses vault, I feel like our documentation on legalities can be better explained. 
4. I think we may be at the point where we can rename the project and focus from SRD to something else. Since not everything is technically an SRD. 

I have it slated that during Memorial Day weekend is when I prepare the winner's tools for June (and if its XWN, it'll be June AND July because its gonna be BOTH worlds), and at that time is when I will want to have these decided upon so we're good to go for the next project and foreseeable future.

**Reply by sigrunixia** [reply]
## Rule and Guidelines
As every SRD and resource is going to be different, there are only a few suggested guidelines on how the conversions and projects should go. Due to legal requirements, there are also a few rules that must be insisted on.

### Rules
1. The content must be legally allowed to be distributed freely by the authors and publishers. Whether it ends up being hosted on [ObsidianTTRPGShare](https://github.com/ObsidianTTRPGProject/ObsidianTTRPGShare) or in your own repositories, if you are working as a part of this project, the content you are working on in the conversion must be a a part of the "free to distribute content." Once the conversion is labeled as "Completed". You are free to do and what you want with your own copy. 
7. Licenses that are required to remain with the content, must be included with the content, including the vault. The repository Read.me is not enough. However, combining all of the legal notices onto one formatted note with headers is appropriate. 
8. If you can find an SRD on Github that is in another format, and can confirm it matches the official SRD version, and follow the licensure requirements above, **you may convert** from that format to Markdown. I.E., from JSON to Markdown. 
9. You **may include** other "free to distribute" content from other publishers for that conversion within the SRD Vault, if it is specifically marked as such, and is separated out into root folders marked by that publisher, and includes licensing attributes. For example, Kobold Press and DND-5e. 

### Guidelines
1. Create all callouts, headings, file structures, tag structures, YAML, and appearance changes with the defaults themes and plugins in mind. 
2. However, work with what Obsidian does have. It does have MOCs, tagging, linking, and great folder management. Try to incorporate all of them.
3. Match the file structure as close to the original SRD as possible. For example, if a PDF has a top-level section called `Playing the Game`, with sub-sections and notes, one of which is named `Combat` with more notes under that, you will want to match that structure with Folders. Your Vault will have a `Root Folder` named `Playing the Game` and inside that will also be a folder called `Combat`.
4. Set your "**Converting Vault**" to use Markdown links, as Github will not accept `Wikilinks`. Additionally, set your "**New Links**" to be "**Relative to Path in File.**"
5. Place items, skills, beasts, classes, actions, conditions, etc., on their own notes. It helps later on when the player or game master decides to use plugins, needs to link, or needs to move things around. 
6. Items, Skills, Stats, Spells, Monsters, and links to other files should be linked **_where it makes sense_**. Not every word needs to be linked to the appropriate skill within the vault, but think about what would be helpful to you as a player, and as a game master. 

---
| Can Use a Link or Two | Can Do Without or Minimal Use |
|:---:|:---:|
|**Being Prone**: When you are **Prone**, you can't use your **Move Action** until you use the **Get Up Action**.| Not everything on **The Street** is determined with fists or guns. In a world where combat can end your life in a hot nanosecond, other methods have evolved to determine who is going to be the top dog in a potential conflict.|
---

7. A certain amount of inclusion of qualify of life is nice, but you do not have to do this. For instance, replacing d6 with `dice: d6` tells the same story, but also makes it so when [Dice Roller](https://github.com/valentine195/obsidian-dice-roller) is installed, that note is ready to go. Similarly, naming a note in the `Combat` folder Combat and having it also be the Map of Content or Table of Contents, makes it so that if someone installs a folder-note plugin, its all done for them. 

10. Last but not least. Think about what you wish you had when you started Obsidian as a resource. What were your pain points when you incorporated your own data into the vault? Can you address this in this SRD? If you downloaded the [DND SRD](https://github.com/OldManUmby/DND.SRD.Wiki), what would you have liked out of it? What would you have done better?

Thank you current and future volunteers. This community is awesome, and we are only just getting started.

**Comment by Craftidore** [comment]
### Re 1: 

What would be the scenario where we want to reword text? Or are we required to reword the text in question?

### Re 2:
 
I believe wikilinks should be used over markdown links, as this is the *Obsidian* TTRPG Community, and wikilinks are first class citizens in Obsidian (and as much as it pains me to say it, markdown links are not given the same prioritization). When we complete an SRD, we can always use [obsidian export](https://github.com/zoni/obsidian-export) to create an alternate version using markdown links, and keep that as a separate branch which we can point people to if they need a markdown SRD for non-Obsidian use. 

> Much of the feedback is coming from "newbies", particularly those with windows systems.

I'm not quite sure how this relates to the question of relative markdown links vs wikilinks; why would OS matter here? Paths are handled correctly regardless of `/` or `\` as I recall.

### Re 3: 

I don't disagree.

### Re 4:

Where do we have the 'project and focus' spelled out? I've checked over the homepage README and it doesn't seem exclusively SRD-focused. 
That said, we are doing more than just SRD conversion, and I approve expanding our official scope to reflect that.

**Reply by sigrunixia** [reply]
For number 1, and @Yilmas can confirm if this is still the case, Mr. Crawford and his publishing entity disallow copying text word-for-word. However, he does allow rewriting the text. So in the case of XWN vaults, we would absolutely be required to reword the text if we wanted to have it public facing. 

2. "I'm not quite sure how this relates to the question of relative markdown links vs wikilinks; why would OS matter here? Paths are handled correctly regardless of / or \ as I recall."

This is more inferring that the technical levels needed to run Windows are not as high as *NIX. macOS could be lumped in there, but tech enthusiasts tend to be drawn towards macOS so its less of an issue. The problem comes when a runs into an issue with breaking the markdown links. It's a stronger likelihood the know-how to fix it without extensive direction is not there. 

4. It used to be on the main TTRPGShare readme, and now it's kinda in the internal readme, but we have gone past that now. :)

**Reply by Yilmas** [reply]
That is correct, any content published by Sine Nominee Inc. (Kevin Crawford) must be reworded, and may only be "copied" from the free edition, this applies to SWN, WWN, and soon CWN. 
With that said, Kevin Crawford, as part of the kickstarter for CWN reached the goal where he would formulate his own "license" for the community. So everybody is waiting for that to reach the public. This is in reference and relation to OGL/paizo-license-thingy... The license should hit sometimes around autumn this year.

**Reply by sigrunixia** [reply]
As an update: 

**Number 1**: There is so much text in Without Numbers.. I think this will need to be covered in license as well as readme and issue. Part of that being, I will need to include something that states throwing it into ChatGPT is not acceptable. There is also the matter of how much we can get away with and claim fair use. Some small sections are okay, despite what Mr. Crawford states. We'll respect his wishes as best he can, but we will come up against things that will not make sense rewritten, or would fundamentally change the rules.

To cover our butts, I will update the SRD language, project scope, and other documents this weekend to indicate that the primary purpose of these conversions is educational in nature, which for me it is. I want to increase access to these worlds that are not affordable to everyone. How can people learn to write stories, and create their own mythos, if that knowledge is locked behind paywalls that are very expensive? It's going to be a fine line, and I hope Mr. Crawford recognises that in his license update in the fall. 

**Number 2**: Cypher and OSE are starting with the new system of Wikilinks. Without Numbers will as well.. It's easy enough to convert wikilink to markdown, but not the other way. We'll need to include a caveat that if people put multiple systems in one vault, links may get mislinked overwritten.

**Number 3**: I think we'll end up rewriting as we work on the Without Numbers System. I _think_ I have it narrowed down close to how it would look in [Old School Essentials Markdown License](https://github.com/Obsidian-TTRPG-Community/Old-School-Essentials-Markdown/blob/releases/License.md).

**Number 4**: Any project name suggestions?
